### PR TITLE
chore: add gitignore for build file

### PR DESCRIPTION
**Pull request by kachick**
`make build` generates this file

https://github.com/google/yamlfmt/blob/569994bdbd896057ca0d60f4efcdb3c8d180a362/Makefile#L2-L3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,5 +5,8 @@ tmp
 # Goreleaser build folder
 dist/
 
+# build file with `make build`
+yamlfmt
```
Comment (braydonk) [inline .gitignore:9]: Might as well add this for Windows while we're at it

```suggestion
yamlfmt
yamlfmt.exe
```

Comment (kachick) [reply .gitignore:9]: Good point👍 
Applied the commit as https://github.com/google/yamlfmt/pull/120/commits/16e74b7bf0ca8786e6ecf9ca57b99e542dd5da58

**Comment by braydonk** [comment]
Thanks looks good!

**Comment by kachick** [comment]
Thank you!

**Pull request merged**